lineman/app/components/thread_page/activity_card/activity_card.scss

```diff
@@ -6,4 +6,14 @@
   @include metaText;
   line-height: 30px;
   span{text-decoration: underline;}
-}
\ No newline at end of file
+}
+
+.activity-card-last-read {
```

Comment:
Hey James.. I'm trying to move the SCSS to pretty strict BEM syntax. Here Block is activity-card and Element is last-read.. so should be separated with double underscore. Just a note for next time.